Libraries/src/net/relinc/libraries/application/Bar.java

```diff
@@ -16,7 +16,7 @@
 	public double diameter;
 	public double speedLimit;
 	public double yield;
-	private double poissonsRatio;
+	private double poissonsRatio=.33;
```

Comment:
This a speculative code change for safety or was this a correction of incorrect behavior? Not opposed to the default value, just curious

Reply:
I think it came out to zero once when I was using it on old bar set-ups.

Reply:
It is a default, but generally close.